delete-friend.php

```diff
@@ -1,6 +1,20 @@
 <?php
-	require "db.php";
-	// TODO: Test
-	$db->query("DELETE FROM friend WHERE username = ? AND friend = ?", [$_SESSION["username"], $_GET["friend"]]);
-	header("Location: home.php");
+	require_once "db.php";
+
+	$error = false;
+	if (isset($_GET["friend"])) {
+		$results = $db->query("select * from friendship where friend = ? and user=?", [$_GET["friend"], $_SESSION["username"]]);
```

Comment:
Not that it matters, but the order of the parameters here is opposite of the delete query and the table